src/pages/about.tsx

```diff
@@ -26,6 +27,8 @@ const AboutUs = ({ lang, textContent, footerLang, navbarLang, metatagsDescriptio
         {sm_breadcrumb('About', 'about')}
       </Script>
 
+      <AhrefsAnalytics lang={lang} />
```

Comment:
If you're planning to add this component to all the pages throughout the project, wouldn't it be better to include it in `/src/pages/_app.tsx` so it's shared across all pages directly?
What do you think?